library/src/main/java/br/com/goncalves/pugnotification/notification/Custom.java

```diff
@@ -24,10 +24,19 @@
     private int mBackgroundResId;
     private int mPlaceHolderResourceId;
     private ImageLoader mImageLoader;
-
-
-    public Custom(NotificationCompat.Builder builder, int identifier, String title, String message, Spanned messageSpanned, int smallIcon, String tag) {
+    private final boolean mUseSpanForCustomNotification;
```

Comment:
Can you please remove the `m` prefix from all variables?

Reply:
Ok. I wasn't sure what approach we want to take as this is a fork of the existing library